### PR TITLE
#99185 Deprecate connector framework

### PR DESCRIPTION
**Pull request by Klaas-Ritense**

### CODE REVIEW

File: plugin/src/main/kotlin/com/ritense/plugin/service/PluginService.kt [hunk 1]
```diff
@@ -536,6 +536,14 @@ class PluginService(
         return pluginConfiguration?.let { createInstance(it) as T }
     }
 
+    fun <T> createInstances(clazz: Class<T>): List<T> {
```
Comment (theo-ritense) [inline plugin/src/main/kotlin/com/ritense/plugin/service/PluginService.kt]: Why was this added if it's not used? And if it's needed: can we keep the filter argument in?

Comment (Klaas-Ritense) [reply plugin/src/main/kotlin/com/ritense/plugin/service/PluginService.kt]: Changed

**Review by theo-ritense — APPROVED** [review]
